introduction.md

```diff
@@ -3,3 +3,8 @@ I'm cromatdragon
 when I was young, I had less years old.
 
 JavaScript
+
+* Javascript
+* Python
+* jQuery
+* HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?